src/users/models/add_human_user_request.rs

```diff
@@ -0,0 +1,250 @@
+/*
+ * User Service
+ *
+ * This API is intended to manage users in a ZITADEL instance.
+ *
+ * OpenAPI spec version: 2.0
+ * Contact: hi@zitadel.com
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+use serde::{Deserialize, Serialize};
+#[allow(unused_imports)]
+use serde_json::Value;
+
+use crate::users::models;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AddHumanUserRequest {
```

Comment:
Should probably use a macro to reduce the boilerplate here, and give the struct a different name, but looks good otherwise.